DEVELOPMENT.md

```diff
@@ -129,23 +121,14 @@ can easily [clean your cluster up](#clean-up) and try again.
 
 ### Setup cluster admin
 
-Your `$K8S_USER_OVERRIDE` must be a cluster admin to perform the setup needed
-for Knative.
-
-The value you use depends on
+The `--user` you use depends on
```

Comment:
Sorry, you give the changes here is covered by my incorrect operation. @adrcunha

Reply:
If you didn't agree with my changes, you're free to do it any other way. But it would be good to at least mention the right value for minikube for clarity.

Reply:
I agree with you changes, but I use the `git push -f` command and forget pull the latest change.  I don't know how to recover it. Could you please help to recover your changes. Thank you! @adrcunha

Reply:
You force pushed your changes, I don't think there's a way for me to view the commits you overwrote. I'll just write it again.